config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :users, only: [ :index, :show ] do
+    resources :posts, only: [ :index, :show ]
+    end
+    
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] [optional] @sunga12 It would have been nice adding a root route in your routes.

Reply:
Thanks for the great suggestion, I have made the changes! 👍